tests/hipscat/io/test_write_metadata_cloud.py

```diff
@@ -0,0 +1,207 @@
+"""Tests of file IO (reads and writes)"""
+
+import os
+
+import hipscat.io.write_metadata as io
+import hipscat.pixel_math as hist
+import numpy.testing as npt
+import pyarrow as pa
+import pytest
+from hipscat.catalog.catalog_info import CatalogInfo
+from hipscat.io import file_io
+from hipscat.io.parquet_metadata import write_parquet_metadata
+
+from hipscat_cloudtests import TempCloudDirectory, assert_text_file_matches
+
+
+@pytest.fixture
+def basic_catalog_parquet_metadata():
+    return pa.schema(
+        [
+            pa.field("id", pa.int64()),
+            pa.field("ra", pa.float64()),
+            pa.field("dec", pa.float64()),
+            pa.field("ra_error", pa.int64()),
+            pa.field("dec_error", pa.int64()),
+            pa.field("__index_level_0__", pa.int64()),
+        ]
+    )
+
+
+@pytest.fixture
+def catalog_info_data() -> dict:
+    return {
+        "catalog_name": "test_name",
+        "catalog_type": "object",
+        "total_rows": 10,
+        "epoch": "J2000",
+        "ra_column": "ra",
+        "dec_column": "dec",
+    }
+
+
+@pytest.fixture
+def catalog_info(catalog_info_data) -> CatalogInfo:
+    return CatalogInfo(**catalog_info_data)
```

Comment:
Could these be moved to a `conftest.py` file so they can be reused later if needed?